### PR TITLE
dev-python/nextinspace: add 3.0.0

### PR DESCRIPTION
**Pull request by hamarituc**
version bump

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: dev-python/nextinspace/nextinspace-3.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+LICENSE="GPL-3+"
+SLOT="0"
+
+RDEPEND="
+	>=dev-python/colorama-0.4.3[${PYTHON_USEDEP}]
+	>=dev-python/requests-2.24[${PYTHON_USEDEP}]
+"
+BDEPEND="
+	test? (
+		${RDEPEND}
```
Comment (mgorny) [inline dev-python/nextinspace/nextinspace-3.0.0.ebuild:30]: That's added by `distutils_enable_tests` automatically.

File: dev-python/nextinspace/nextinspace-3.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	)
+"
+
+distutils_enable_tests pytest
+
+src_prepare() {
+	default
+
+	# Don't install license files
+	sed -e '/^include = \["LICENSE"\]$/d' -i pyproject.toml || die
```
Comment (mgorny) [inline dev-python/nextinspace/nextinspace-3.0.0.ebuild:42]: Why isn't there a pull request upstream to fix this?

**Review by mgorny — CHANGES_REQUESTED** [review]

File: dev-python/nextinspace/nextinspace-3.0.0.ebuild (hunk 1 above)
Comment (hamarituc) [reply dev-python/nextinspace/nextinspace-3.0.0.ebuild]: fixed

File: dev-python/nextinspace/nextinspace-3.0.0.ebuild (hunk 2 above)
Comment (hamarituc) [reply dev-python/nextinspace/nextinspace-3.0.0.ebuild]: I wasn't aware the issue behind that fix is a general type of error. I submitted a fix to upstream and learned a bit more about python packages by the way :smiley:

Comment (mgorny) [reply dev-python/nextinspace/nextinspace-3.0.0.ebuild]: Wait, you've never seen my "Poetry is a true horror" series of toots? Perhaps it's time to put them in a blog post.

**Review by mgorny — APPROVED** [review]
Thanks!

**Pull request closed without merging**